frontend/src/components/plan-components/Stundenplan.tsx

```diff
@@ -232,20 +255,20 @@ export default function Stundenplan(): JSX.Element {
     return(
         <div className="table-layout">
             <div className="table-top">
-                <span className="day">{currentDates[0]}<br/>Mo.</span>
-                <span className="day">{currentDates[1]}<br/>Di.</span>
-                <span className="day">{currentDates[2]}<br/>Mi.</span>
-                <span className="day">{currentDates[3]}<br/>Do.</span>
-                <span className="day">{currentDates[4]}<br/>Fr.</span>
+                <span className="day" style={dateStyle[1]}>{currentDates[0]}<br/>Mo.</span>
```

Comment:
dude, don't use a style attribute, just add a className. Then you can also just add/remove that className to the relevant elements instead of having an array for this